src/chainparams.cpp

```diff
@@ -190,11 +188,11 @@ class CTestNetParams : public CMainParams {
 
         //! Modify the testnet genesis block so the timestamp is valid for a later start.
         genesis.nTime = 1296688602;
```

Comment:
This timestamp is 2011-02-02 23:16:42. Does not block (we're resetting again, I think).